File: 2.0/java/campaigns/updateCampaignBatch.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.campaigns.model.Batch;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        Batch batch = client.batchesApi().get(379506003L);
        batch.setEnabled(true);
        client.batchesApi().update(batch);
    }
}

File: 2.0/csharp/campaigns/updateCampaignBatch.cs
using CallfireApiClient;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var batch = client.BatchesApi.Get(379506003);
        batch.Enabled = true;
        client.BatchesApi.Update(batch);
    }
}
